food-truck-frontend/src/api/Notification.ts

```diff
@@ -0,0 +1,56 @@
+import api from "../util/api";
+import {TruckState} from "../components/TruckView";
+
+export interface Notification {
+  id: number;
+  message: string;
+  truck: TruckState;
+  time: Date;
+  read: boolean;
+}
+
+export const getNotifications = async () => {
+  let notifications: Notification[] = [];
+
+  await api.get(`/notifications`, {})
+    .then((response: any) => {
+      if (response.data) {
+        notifications = response.data;
+      }
+    });
+
+  return notifications;
+}
+
+export const sendNotification = async (
+  truckId: number,
+  message: string,
+  onFail?: (res: any) => void
+) => {
+  let config = {
+    truckId: truckId,
+    message: message
+  };
+  await api.post(`/truck/notification`, config)
```

Comment:
Never seen using `post()` before, I always just configured it manually (arguably, yours is probably more correct... :P). I'm guessing config is then passed as the request body, or that Spring auto-wraps the data into the parameter class in the backend?

Reply:
Yep. The config is the request body.